tests/vmi_lifecycle_test.go

```diff
@@ -686,23 +686,15 @@ var _ = Describe("VMIlifecycle", func() {
 			})
 
 			It("should enable emulation in virt-launcher", func() {
-				err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(tests.NamespaceTestDefault).Body(vmi).Do().Error()
-				Expect(err).To(BeNil(), "Should post VMI successfully")
+				vmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
 
-				listOptions := metav1.ListOptions{}
-				var pod k8sv1.Pod
+				tests.WaitForSuccessfulVMIStart(vmi)
 
-				Eventually(func() error {
-					podList, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).List(listOptions)
-					Expect(err).ToNot(HaveOccurred(), "Should list pods")
-					for _, item := range podList.Items {
-						if strings.HasPrefix(item.Name, vmi.ObjectMeta.GenerateName) {
-							pod = item
-							return nil
-						}
-					}
-					return fmt.Errorf("Associated pod for VirtualMachineInstance '%s' not found", vmi.Name)
-				}, 75, 2).Should(Succeed(), "Should find the right VMI pod")
+				pods, err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).List(tests.UnfinishedVMIPodSelector(vmi))
```

Comment:
Sure, I just did not find it:( because `Code` IDE does not have the possibility to list all methods under the file and our `utils.go` grows each time.